README.md

```diff
@@ -96,6 +96,53 @@ $app->post('/api/myEndPoint',function ($req, $res, $args) {
 $app->run();
 ```
 
+### Accessing the token pair in templates (Twig, etc)
+
+In many situations, you will want to access the token pair without needing to go through the request object.  In these cases, you can use `getTokenName()` and `getTokenValue()` directly on the `Guard` middleware instance.  This can be useful, for example in a [Twig extension](http://twig.sensiolabs.org/doc/advanced.html#creating-an-extension):
+
+```php
+class CsrfExtension extends \Twig_Extension
+{
+
+    /**
+     * @var \Slim\Csrf\Guard
+     */
+    protected $csrf;
+    
+    public function __construct(\Slim\Csrf\Guard $csrf)
+    {
+        $this->csrf = $csrf;
+    }
+
+    public function getGlobals()
+    {
+        // CSRF token name and value
+        $csrfNameKey = $this->csrf->getTokenNameKey();
+        $csrfValueKey = $this->csrf->getTokenValueKey();
+        $csrfName = $this->csrf->getTokenName();
+        $csrfValue = $this->csrf->getTokenValue();
+        
+        return [
+            'csrf'   => [
+                'keys' => [
+                    'name'  => $csrfNameKey,
+                    'value' => $csrfValueKey
+                ],
+                'name'  => $csrfName,
+                'value' => $csrfValue
+            ]
+        ];
+    }
```

Comment:
In the meantime, things have changed:
```php
/*
 * @deprecated since 1.26 (to be removed in 2.0), not used anymore internally
 */
function getName();
```

But `getName` does not hurt and must be implemented to support older versions.